spec/factories/uploaded_file_factory.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :uploaded_file do
+    # Adding this line to suppress rubocop warning about empty block
+  end
```

Comment:
Tiny nit, you don't even need to pass the block here. Just `factory :uploaded_file` should work.